packages/blockchain-wallet-v4-frontend/src/components/Flyout/InterestUploadDocuments/Uploaded.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { IntlProvider } from 'react-intl'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import Uploaded from './Uploaded'
+
+export default {
+  argTypes: {},
+  args: {
+    close: () => {}
+  },
+  component: Uploaded,
+  decorators: [
+    (Story) => (
+      <IntlProvider locale='en'>
+        <div style={{ display: 'flex', height: '100vh', width: '480px' }}>
+          <Story />
```

Comment:
Sometimes doing `<Story />` makes it preform in weird ways. I usually use `{Story()}` instead.